editor/configs/nonCR3Columns.ts

```diff
@@ -0,0 +1,26 @@
+import { ColDataCardDef } from "@/types/types";
+import { NonCR3Record } from "@/types/nonCr3";
+
+export const nonCR3Columns = {
+  address: {
+    path: "address",
+    label: "Address",
+    sortable: true,
+  },
+  case_timestamp: {
```

Comment:
wold you add a formatter to this so that it renders in the same format as the crash date and incident date?